From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Mon, 11 Jul 2022 21:28:57 +0900
Subject: Debian uses "pycryptodomex" instead of "prcryptodome"

Forwarded: not-needed

Debian uses Cryptdome class (pycryptodomex) for cipher functions.
Cryptodome class can be replacement for old Crypto class (prcryptodome),
but Debian dose not replace this class yet.

So, this hack is not needed yet.
---
 src/calibre/constants.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/calibre/constants.py b/src/calibre/constants.py
index fcd83b9..b0a2723 100644
--- a/src/calibre/constants.py
+++ b/src/calibre/constants.py
@@ -215,10 +215,6 @@ class DeVendor:
             if fullname == 'PyQt5':
                 return ModuleSpec(fullname, DeVendorLoader('qt'))
             return ModuleSpec(fullname, DeVendorLoader('qt.webengine' if 'QWebEngine' in fullname else 'qt.core'))
-        if fullname.startswith('Cryptodome'):
-            # this is needed for py7zr which uses pycryptodomex instead of
-            # pycryptodome for some reason
-            return ModuleSpec(fullname, DeVendorLoader(fullname.replace('dome', '', 1)))
 
 
 class ExtensionsPackageLoader:
